### DISCUSSION TITLE
FileTreeView for the filesystem

### DISCUSSION THREAD

**Discussion by slamcoderpt** [filamentphp/filament#14078] [Ideas]
What about a file tree view  that uses the disk / filepath to see in a tree view the files and folders inside. Also could have actions like open and delete (file/folder) and custom actions.
Something like this: https://www.jstree.com/

**Comment by sinnbeck** [comment]
This might satisfy your needs?

https://filamentphp.com/plugins/3x1io-tomato-browser

**Reply by slamcoderpt** [reply]
Yes thank you!
